packages/react/public/types.d.ts

```diff
@@ -47,6 +50,7 @@ export type OidcConfig = {
     oidcConfiguration?: OidcConfiguration;
     sessionState?: string | null;
     items?: MessageData;
+    hideAccessToken: boolean;
```

Comment:
`hideAccessToken` is `true` by default. Should we call this the opposite: `showAccessToken`?